db/import.go

```diff
@@ -256,7 +255,7 @@ func (db *Database) migrateMetadata(docid string, body Body, existingDoc *sgbuck
 	// TODO: Could refactor migrateMetadata to use WriteUpdateWithXattr for both CAS retry and general write handling, and avoid cast to CouchbaseBucketGoCB
 	gocbBucket, ok := db.Bucket.(*base.CouchbaseBucketGoCB)
 	if !ok {
-		return nil, false, fmt.Errorf("Metadata migration requires gocb bucket (%T)", db.Bucket)
+		return nil, false, base.RedactErrorf("Metadata migration requires gocb bucket (%T)", base.MD(db.Bucket))
```

Comment:
Here we're just getting the type of `db.Bucket`, and not the actual value. I don't think we want to redact this particular instance.